CREDITS.md

```diff
@@ -12,10 +12,11 @@ https://github.com/pbering/SockerVS2017
 https://github.com/pbering/solrwin
 
 **Joost Meijles (Aviva Solutions)**: His contributions in GitHub have been a source of inspiration. He is also active in community discussions about using Docker with Sitecore.
```

Comment:
Please add Jonne Kats (he is the Jonne below) here as well. His efforts are a big as mine for the avivasolutionsnl docker repos. Thanks.

Reply:
Done